daemon/src/maker_cfd.rs

```diff
@@ -200,7 +200,7 @@ impl Actor {
             oracle::next_announcement_after(time::OffsetDateTime::now_utc() + Order::TERM)?;
 
         self.oracle_actor
-            .do_send_async(oracle::FetchAnnouncement(oracle_event_id.clone()))
+            .do_send_async(oracle::FetchAnnouncement(oracle_event_id))
```

Comment:
❓ Just noticed that now - not related to this PR - what is the difference between `FetchAnnouncement` and `GetAnnouncement` ? Is this in the literal sense - i.e. `GetAnnouncement` (tries to) get it from the locally known state and `FetchAnnouncement` fetches it from the remote?

Reply:
I will add documentation.